omero/sysadmins/unix/server-centos7-ice36.rst

```diff
@@ -10,6 +10,8 @@ a dedicated local system user, and should be read in conjunction with
 for setting up your own test server. For production use you should also read
 the pages listed under :ref:`index-optimizing-server`.
 This guide will install Python 3.6.
+Since 5.6, a new ``OMERODIR`` variable is used, you should first unset ``OMERO_HOME``
```

Comment:
```suggestion
Since 5.6, a new :envvar:`OMERODIR` variable is used, you should first unset :envvar:`OMERO_HOME`
```

Reply:
I guess the same fix should be applied to the other places too?
Otherwise looks good.